init.lua

```diff
@@ -157,68 +151,40 @@ minetest.register_node("flowerpot:empty", {
 		fixed = {-1/4, -1/2, -1/4, 1/4, -1/16, 1/4},
 	},
 	sounds = default.node_sound_defaults(),
-	groups = {attached_node = 1, oddly_breakable_by_hand = 3, cracky = 1, dig_immediate = 3},
-	on_rightclick = flowerpot_on_rightclick,
+	groups = {attached_node = 1, oddly_breakable_by_hand = 3, cracky = 1},
+
+	on_rightclick = flowerpot_on_rightclick
 })
 
--- craft
+-- Craft
 minetest.register_craft({
 	output = "flowerpot:empty",
 	recipe = {
 		{"default:clay_brick", "", "default:clay_brick"},
-		{"", "default:clay_brick", ""},
+		{"", "default:clay_brick", ""}
 	}
 })
 
--- default farming nodes
+-- Groupable nodes
+local nodes = {}
+for node, def in pairs(minetest.registered_nodes) do
+	if def.groups.flora == 1 or def.groups.sapling == 1 or def.groups.plant == 1
+	or node:find("mushroom") or node:find("stem") then
```

Comment:
This will match all nodes registered within https://github.com/TalkLounge/plot_system 
Great job, but please make these exceptions a separate hard-coded list to not get false-positives.

Reply:
I didn’t quite understand which of my filters might not work as expected. Clarify, please.

Reply:
The issue is that string searching in node names is not reliable. You only want plants to have a flowerpot.
Nodes like `mymod:system_fan_controller` or `itemstemperature:foobar` would get a flower pot registration. That's definitely not intended.

Reply:
You're right. I will come up with a more reliable solution. Thanks for the review.

Reply:
@SmallJoker 
...**or string.sub(node, -5)== "_stem"**

Reply:
That's already better, although adding a new group to the node definition (MTG change) would be the best solution.

I think for now it's okay to use `string.sub(node, -5)== "_stem"` even if there might still be rare false-positives.

Reply:
Fixed

Reply:
I really don't like this wildcard method. You can trivially create e a secondary mod that adds registrations for wildcard nodes instead. I prefer much to not have a wildcard instead.